src/Synthesiser.cpp

```diff
@@ -2339,17 +2237,6 @@ void Synthesiser::generateCode(std::ostream& os, const std::string& id, bool& wi
         os << R"_(souffle::ProfileEventSingleton::instance().makeConfigRecord("version", ")_"
            << Global::config().get("version") << R"_(");)_" << '\n';
     }
-#ifdef USE_MPI
-    if (Global::config().get("engine") == "mpi") {
-        os << "\n#ifdef USE_MPI\n";
-        os << "souffle::mpi::init(argc, argv);";
-        os << "int rank = souffle::mpi::commRank();";
-        os << "int stratum = (rank == 0) ? " << std::numeric_limits<int>::max() << " : rank - 1;";
-        os << "obj.runAll(opt.getInputFileDir(), opt.getOutputFileDir(), stratum);\n";
-        os << "souffle::mpi::finalize();";
-        os << "\n#endif\n";
-    } else
-#endif
     {
```

Comment:
These braces are no longer required. They were only needed to enable the, now removed, if else stucture.